src/codec.js

```diff
@@ -40,7 +40,8 @@ function stringToStringTuples (str) {
     var part = parts[p]
     var proto = protocols(part)
     if (proto.size === 0) {
-      return [part]
```

Comment:
this was a previously never caught bug, since all the transports it supported before always had an addr of size > 0